ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -25,11 +25,12 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 message Event {
   oneof event {
     CreatedEvent created = 1;
+    ExercisedEvent exercised = 2;
     ArchivedEvent archived = 3;
  }
-  // see https://github.com/digital-asset/daml/issues/960
-  reserved 2;
-  reserved "exercised";
+//  // see https://github.com/digital-asset/daml/issues/960
```

Comment:
Why keep the comment here?

Reply:
I'd suggest to add commentary on the field referencing #960 and explaining why it was re-introduced.